resources/language/resource.language.hu_hu/strings.po

```diff
@@ -8,13 +8,8 @@ msgstr ""
 "Report-Msgid-Bugs-To: https://github.com/CastagnaIT/plugin.video.netflix\n"
 "POT-Creation-Date: 2017-07-24 16:15+0000\n"
 "PO-Revision-Date: 2020-09-05 07:40+0000\n"
-"Last-Translator: frodo19"
-"Language-Team: Hungarian\n"
-"MIME-Version: 1.0\n"
-"Content-Type: text/plain; charset=UTF-8\n"
-"Content-Transfer-Encoding: 8bit\n"
-"Language: hu\n"
-"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+"Last-Translator: frodo19\n"
+"Language-Team: Hungarian[CR]MIME-Version: 1.0[CR]Content-Type: text/plain; charset=UTF-8[CR]Content-Transfer-Encoding: 8bit[CR]Language: hu[CR]Plural-Forms: nplurals=2; plural=(n != 1);[CR]\n"
```

Comment:
only a minor thing here the original header metadata is broken and some are missing

Reply:
Ow. Will fix this. I thought I had restored all headers.

Reply:
Fixed!